package.json

```diff
@@ -57,7 +57,7 @@
     "test:coverage": "nyc --silent qunit test/node_test_setup.js test/lib test/unit",
     "test:visual:coverage": "nyc --silent --no-clean qunit test/node_test_setup.js test/lib test/visual",
     "coverage:report": "nyc report --reporter=lcov --reporter=text",
-    "lint": "eslint --config .eslintrc.js src/**/*.ts",
+    "lint": "eslint --config .eslintrc.js src",
```

Comment:
this was my issue. 
i was getting only files inside the first folders.
VSCode indeed was looking and eslint.rc and workin correctly.